libs/core/src/lib/nested-list/nested-list-directives.ts

```diff
@@ -93,9 +93,13 @@ export class NestedListTitleDirective {
 export class NestedListExpandIconDirective {
 
     /** @hidden */
-    @HostBinding('class.fd-nested-list__expand-icon')
+    @HostBinding('class.fd-nested-list__button')
     fdNestedListTitleClass = true;
 
+    /** @hidden */
+    @HostBinding('class.fd-button')
+    fdButtonClass = true;
+
```

Comment:
That's the thing we need to test in separated app, if it will work with only side nav, without button component import